pyproject.toml

```
@@ -15,22 +15,24 @@ authors = [
 ]
 description = "Fast asteroid precovery at scale"
 readme = "README.md"
-license = { file = "LICENSE.md", classifier = "BSD License" }
+license = { file = "LICENSE.md"}
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: BSD License",
     "Operating System :: OS Independent",
-    "Programming Language :: Python",
     "Programming Language :: Python :: 3.10",
     "Topic :: Scientific/Engineering :: Astronomy",
     "Topic :: Scientific/Engineering :: Physics",
 ]
 
 dependencies = [
-    "adam_core>=0.2.0",
+    "adam_core[assist]>=0.2.0",
+    "adam_assist",
```

```suggestion
    "adam_core>=0.2.3",
```

Let's move `adam-assist>=0.1.2` to the dev optional-dependencies. Also let's require more recent adam_core.